### DISCUSSION TITLE
JBrowseLinearGenomeView height

### DISCUSSION THREAD

**Discussion by yonglbl** [GMOD/jbrowse-components#4650] [Q&A]
Is there a way to set the browser height of JBrowseLinearGenomeView ?

**Comment by cmdcolin** [comment]
Hi there,
Currently there is not really a way to do this @jbrowse/react-linear-genome-view currently, but there is a feature request here for it
https://github.com/GMOD/jbrowse-components/issues/4526

note that the @jbrowse/react-app actually does do this, and it uses a set height of 100vh (1 screen height).
